fitbenchmarking/fitbenchmarking
# Add SciPy Global Optimization software

#### Description of Work

Closes #822 


#### Testing Instructions

1. Check that all three SciPy GO minimisers run on a simple test problem with bounds
2. Check that docs have been updated to include scipy_go
3. Check that the new global_optimization algorithm type label works correctly
4. Check that the new MissingBoundsError is correctly raised

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
